### ISSUE TITLE
voices are not working correctly in vr mode with new dlc

### ISSUE THREAD

**Issue by theboobness** [Eusth/HoneySelectVR#54]
Hi Eusth, the voices aren't working correctly when I go in an H scene. there would be multiple voices (4 or more) playing at the same time. In normal mode without vr, the game plays fine. Is there any way to fix this?

**Comment by Eusth** [comment]
Completely untested, but try this: [HoneySelectVR_0.6.1.zip](https://github.com/Eusth/HoneySelectVR/files/966658/HoneySelectVR_0.6.1.zip)

**Comment by theboobness** [comment]
Thank you!! it works perfectly now!!

**Comment by plasticfant** [comment]
I posted this to the honeyselect subreddit so more VR users can test.
https://www.reddit.com/r/HoneySelect/comments/6930pt/anyone_able_to_get_vr_working_with_the_party_dlc/

As always, thank you Eusth for your huge contribution to VR

**Comment by deinlandel** [comment]
@Eusth Could you please tell what did you patched? I use custom HoneySelect plugin built from sources, and it seems that these changes are not published on Github...

**Comment by Eusth** [comment]
Yup, because it breaks the positional audio. Not gonna add the "fix" to the code. Simply remove the VoiceShim line in HoneySelectVR.cs.

**Comment by Devilsbane99** [comment]
Thank you so much for this Eusth! Working great!

**Comment by ice36bc** [comment]
Thanks this the sound is working right now.

**Comment by gzhuji** [comment]
Thanks for update = v =

**Comment by plasmacut** [comment]
Does that mean there is no working positional audio for 1.20? Also my folder does not contain a HoneySelect.cs file. When using VR patch 0.6 there is also no positional audio unless it's used on version 1.1 for me.

**Comment by ZerothAngel** [comment]
The reason positional audio doesn't work in 1.20 is because a new `bool force2D` parameter was added to the `Play` and `OnecePlayChara` methods in `Manager.Voice`. For whatever reason, `OnecePlayChara` is called with this set to true during H-scenes.

I managed to fix this in a quick & dirty way (see my fork), but:
 1. I decided to use Harmony for runtime patching (just to learn it).
 2. I always set `force2D` to false.

**Comment by spammydavis** [comment]
> Completely untested, but try this: [HoneySelectVR_0.6.1.zip](https://github.com/Eusth/HoneySelectVR/files/966658/HoneySelectVR_0.6.1.zip)

can you release the code for 0.6.1?  sit down behaves differently than 0.6 and I'd like to see what you added.  I recompiled the 0.6 so the "filter" included NEO, but it doesn't incorporate the 0.6.1 changes.

**Comment by DragonQuestHero** [comment]
nice work!
